FirebaseSessions/Tests/Unit/SessionStartEventTests.swift

```diff
@@ -183,4 +185,200 @@ class SessionStartEventTests: XCTestCase {
       )
     }
   }
+
+  func test_convertNetworkType_convertsCorrectly() {
+    let expectations: [(
+      given: GULNetworkType,
+      expected: firebase_appquality_sessions_NetworkConnectionInfo_NetworkType
+    )] = [
+      (
+        .WIFI,
+        firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_WIFI
+      ),
+      (
+        .mobile,
+        firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_MOBILE
+      ),
+      (
+        .none,
+        firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_DUMMY
+      ),
+    ]
+
+    expectations.forEach { (
+      given: GULNetworkType,
+      expected: firebase_appquality_sessions_NetworkConnectionInfo_NetworkType
+    ) in
+      appInfo.networkType = given
+
+      let event = SessionStartEvent(identifiers: identifiers, appInfo: appInfo, time: time)
+
+      testProtoAndDecodedProto(sessionEvent: event) { proto in
+        XCTAssertEqual(
+          event.proto.application_info.apple_app_info.network_connection_info.network_type,
+          expected
+        )
+      }
+    }
+  }
+
+  /// Following tests can be run only in iOS environment
+  #if TARGET_OS_IOS
```

Comment:
Same here and for the next test about the ObjC vs Swift macros. I think these tests are failing because of the of the macro issue in `SessionStartEvent.swift`